scripts/ci/run-jenkins.sh

```diff
@@ -108,6 +106,8 @@ ${TESTCMD} --label=make --timeout=${make_timeout} --fatal make ${make_parallelis
 if [[ ${CI_TAGS} == *'checked-coop'* ]]; then export MONO_CHECK_MODE=gc,thread; fi
 if [[ ${CI_TAGS} == *'checked-all'* ]]; then export MONO_CHECK_MODE=all; fi
 
+export MONO_ENV_OPTIONS="$MONO_ENV_OPTIONS $MONO_TEST_FLAGS"
```

Comment:
why do we need another variable? shouldn't MONO_ENV_OPTIONS be enough?

Reply:
MONO_ENV_OPTIONS will affect the Mono build while MONO_TEST_FLAGS should only  affect the tests. That may make sense if you want to use the default options while building mscorlib, but pass some exotic option to the test suite. As one example, building mscorlib seems to fail with MONO_ENV_OPTIONS=--llvm, even though --llvm does work on most benchmarks.

Reply:
OK, makes sense. I'd call it `MONO_TEST_ENV_OPTIONS` though so it's more consistent.